tests/NCronJob.Tests/IntegrationTests.cs

```diff
@@ -663,13 +663,40 @@ public async Task StaticAnonymousJobsCanBeExecutedMultipleTimes()
     }
 
     [Fact]
-    public void AddingJobsWithTheSameCustomNameLeadsToException()
+    public void AddingUntypedJobsWithTheSameCustomNameLeadsToException()
     {
         Action act = () => ServiceCollection.AddNCronJob(
             n => n.AddJob(() => { }, Cron.AtEveryMinute, jobName: "Job1")
                 .AddJob(() => { }, Cron.AtMinute0, jobName: "Job1"));
 
-        act.ShouldThrow<InvalidOperationException>();
+        var e = act.ShouldThrow<InvalidOperationException>();
+        e.Message.ShouldStartWith("Job registration conflict detected. A job has already been registered with the name 'Job1'.");
```

Comment:
I am always not sure if we really should check the message, or if alone checking the type is good enough. But let's role with it.